list_of_prime_numbers/main.py

```diff
@@ -1,10 +1,24 @@
 
 def _is_prime(number):
-    pass
+    number = int(number)
```

Comment:
Same here. No need to pass number to int(). is_prime(number) takes in integers as an arguments.